sql/core/src/main/scala/org/apache/spark/sql/internal/SharedState.scala

```diff
@@ -136,6 +139,21 @@ private[sql] class SharedState(val sparkContext: SparkContext) extends Logging {
     }
     SparkSession.sqlListener.get()
   }
+
+  def addJar(path: String): Unit = {
```

Comment:
Moving this back to SessionState to avoid that someone calls it via SharedState by mistake in future.